statsig_rum/manifest.json

```
@@ -0,0 +1,45 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "86a69c7d-8042-4f93-96b1-f139b2058644",
+  "app_id": "statsig-rum",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
```

Could you add some uninstallation instructions in the readme?

```suggestion
    "configuration": "README.md#Setup",
    "uninstallation": "README.md#Uninstallation",
    "support": "README.md#Support",
```